platform/openharmony/detect.py

```diff
@@ -0,0 +1,180 @@
+import os
+import sys
+from typing import TYPE_CHECKING
+
+from methods import print_error
+from platform_methods import validate_arch
+
+if TYPE_CHECKING:
+    from SCons.Script.SConscript import SConsEnvironment
+
+
+def get_name():
+    return "OpenHarmony"
+
+
+def can_build():
+    return True
+
+
+def get_tools(env: "SConsEnvironment"):
+    return ["clang", "clang++", "as", "ar", "link"]
+
+
+def get_opts():
+    from SCons.Variables import BoolVariable
+
+    return [
+        ("OPENHARMONY_SDK_PATH", "Path to the OpenHarmony SDK", get_default_sdk_path()),
+        BoolVariable(
+            "generate_bundle",
+            "Generate an APP bundle after building OpenHarmony binaries",
+            False,
```

Comment:
If multi-arch bundles are not supported, why not change the default value of `generate_bundle` to True?

Reply:
Done